src/testdir/test_taglist.vim

```diff
@@ -99,8 +99,11 @@ func Test_tagfiles()
   call assert_equal(['Xtags1', 'Xtags2'], tagfiles())
 
   help
-  let tf = tagfiles()
-  call assert_equal(1, len(tf))
+  let hplang = &helplang
+  set helplang=en  " May other 'tags-xx' (e.g 'tags-ru') exist
+  let tf = tagfiles()->filter("v:val =~ 'tags\\%(-' .. &helplang .. '\\)\\?$'")
+  call assert_equal((&helplang != '' && &helplang != 'en') ? 2 : 1, len(tf))
```

Comment:
`&helplang` is always `'en'` here now.

Reply:
yes, i was trying to add 'ru', but failed, see comment here: https://github.com/vim/vim/pull/14566#issuecomment-2058458220